root/wrapper.html

```diff
@@ -79,7 +79,9 @@
         <%- END %>
         <%- IF need_less %>
         <script src="/static/js/less.min.js" type="text/javascript"></script>
+        <link rel="preload" href="/static/fonts/fa-regular-400.woff2" as="font" />
```

Comment:
This shouldn't be inside this if block. I think it also needs the crossorigin attribute to work properly.

Reply:
Thanks, addressed both.